extensions/wikia/EmbeddableDiscussions/templates/DiscussionThreads.mustache

```diff
@@ -33,43 +33,38 @@
 			</div>
 			<div class="embeddable-discussions-post-counters">
 				<span class="embeddable-discussions-post-counter">
-					<svg class="embeddable-discussions-upvote-icon-tiny"></svg>
+					<img class="svg embeddable-discussions-upvote-icon-tiny" src="{{upvoteTinyIconSrc}}" width="14" height="16">
 					{{upvoteCount}}
 				</span>
 				<span class="embeddable-discussions-post-counter">
-					<svg class="embeddable-discussions-reply-icon-tiny"></svg>
+					<img class="svg embeddable-discussions-reply-icon-tiny" src="{{replyTinyIconSrc}}" width="14" height="16">
 					{{commentCount}}
 				</span>
 			</div>
 		</div>
 		<div>
 			<ul class="embeddable-discussions-post-actions">
-				<li class="small-4 large-4 embeddable-discussions-upvote-area">
+				<li class="embeddable-discussions-upvote-area">
 					{{#hasUpvoted}}
 						<a href="{{upvoteUrl}}" class="upvote" data-id="{{firstPostId}}" data-hasUpvoted="1">
-							<svg class="embeddable-discussions-upvote-icon-active"></svg>
+                            <img class="svg embeddable-discussions-upvote-icon-active" src="{{upvoteIconSrc}}" width="24" height="22">
 							{{upvoteText}}
 						</a>
 					{{/hasUpvoted}}
 					{{^hasUpvoted}}
 						<a href="{{upvoteUrl}}" class="upvote" data-id="{{firstPostId}}" data-hasUpvoted="0">
-							<svg class="embeddable-discussions-upvote-icon"></svg>
-							{{upvoteText}}
+                            <img class="svg embeddable-discussions-upvote-icon" src="{{upvoteIconSrc}}" width="24" height="22">
+                            {{upvoteText}}
 						</a>
 					{{/hasUpvoted}}
 				</li>
-				<li class="small-4 large-4 embeddable-discussions-replies-area">
+                <li class="embeddable-action-gap"></li>
```

Comment:
Is something wrong with indentation here? Or is it just github going wild? Also - why do we need it? Seems like this one has no content in it :)